device-types/Fortinet/FG-60D.yml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: Fortinet
+model: FortiGate 60D
+slug: fg-60d
+part_number: FG-60D
+u_height: 1
```

Comment:
It looks like this is not rack mountable, according to the doc for the `60D` the device is desktop or wall-mount. Please make this `u_height: 0`

Reply:
I will confirm with my team how we want to handle this, but this is my general thinking:
If a device is not rack mountable according to the manufacturer, i.e. the manufacturer only lists the devices as `Desktop and wall-mountable` but does not sell a rack mount adapter, then the device should be listed as 0U. During the creation of the device, the user can then change it to 1U if the device does indeed utilize a 3rd part adapter to make it rack mountable

Reply:
Please leave this unresolved until the maintainers come to a consensus. Thank you.

Reply:
So here is what we are going with. The device itself is not rack mountable. Thus in this repo, which is suppose to be a source of truth, this should have `u_height: 0`. Then, if you specifically want to use this as a rack mountable with the third party adapter, you can manually edit the device definition in your NetBox instance.

Reply:
done